tests/app/views/test_briefs.py

```diff
@@ -0,0 +1,471 @@
+                    'lot': 'digital-specialists',
+                    'title': 'the title',
+                },
+                'update_details': {
+                    'updated_by': 'example'
+                }
+            }),
+            content_type='application/json')
+        data = json.loads(res.get_data(as_text=True))
+
+        assert res.status_code == 201
+        assert data['briefs']['frameworkSlug'] == 'digital-outcomes-and-specialists'
+        assert data['briefs']['title'] == 'the title'
+
+    def test_create_fails_if_lot_does_not_require_briefs(self):
+        res = self.client.post(
+            '/briefs',
+            data=json.dumps({
+                'briefs': {
+                    'userId': self.user_id,
+                    'frameworkSlug': 'digital-outcomes-and-specialists',
+                    'lot': 'user-research-studios',
+                    'title': 'the title',
+                },
+                'update_details': {
+                    'updated_by': 'example'
+                }
+            }),
+            content_type='application/json')
+        data = json.loads(res.get_data(as_text=True))
+
+        assert res.status_code == 400
+        assert data['error'] == "Lot 'User research studios' does not require a brief"
+
+    def test_create_fails_if_required_field_is_not_provided(self):
+        res = self.client.post(
+            '/briefs',
+            data=json.dumps({
+                'briefs': {
+                    'userId': self.user_id,
+                    'frameworkSlug': 'digital-outcomes-and-specialists',
+                    'lot': 'digital-specialists',
+                },
+                'update_details': {
+                    'updated_by': 'example'
+                },
+                'page_questions': ['title'],
+            }),
+            content_type='application/json')
+        data = json.loads(res.get_data(as_text=True))
+
+        assert res.status_code == 400
+        assert data['error'] == {'title': 'answer_required'}
+
+    def test_can_only_create_briefs_on_live_frameworks(self):
+        with self.app.app_context():
+            framework = Framework.query.filter(Framework.slug == 'digital-outcomes-and-specialists').first()
+            self._original_framework_status = framework.status
+            framework.status = 'open'
+
+            db.session.add(framework)
+            db.session.commit()
+
+        res = self.client.post(
+            '/briefs',
+            data=json.dumps({
+                'briefs': {
+                    'userId': self.user_id,
+                    'frameworkSlug': 'digital-outcomes-and-specialists',
+                    'lot': 'digital-specialists',
+                    'title': 'the title',
+                },
+                'update_details': {
+                    'updated_by': 'example'
+                }
+            }),
+            content_type='application/json')
+
+        assert res.status_code == 400
```

Comment:
Might be nice to test the error message here too as there may be several reasons for a 400 and it's good to know the failure is for the right reason.